public/static/locales/en/sharing.json

```diff
@@ -1,5 +1,6 @@
 {
     "cannotAddSelf": "Cannot add yourself for sharing",
+    "cyverseId": "CyVerse ID: {{id}}",
```

Comment:
I went with `CyVerse ID:` but perhaps `ID:` or `Username:` would work too?

Reply:
`Username` makes sense.

Reply:
But the search field prompt says `CyVerse ID`. So if we are changing, we should be consistent.